src/test/java/teammates/test/cases/action/InstructorStudentRecordsAjaxPageActionTest.java

```diff
@@ -78,10 +78,18 @@ protected InstructorStudentRecordsAjaxPageAction getAction(String... params) {
         return (InstructorStudentRecordsAjaxPageAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
-    @Override
     @Test
+    @Override
     protected void testAccessControl() throws Exception {
-      //TODO: implement this
+        InstructorAttributes instructor = typicalBundle.instructors.get("instructor3OfCourse1");
+        StudentAttributes student = typicalBundle.students.get("student2InCourse1");
+
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, instructor.courseId,
```

Comment:
Why are you accessing `courseId` from `instructor`?

Reply:
I have seen in above function like line 57 where it is done like this only. Also student has no attribute of courseid

Reply:
It is probably better to retrieve the course from the typicalBundle and use the course id from the course, rather than a random instructor of the course.

Reply:
Have a look